tools/sklearn/test-data/ohe_out_5.tsv

```diff
@@ -0,0 +1,8 @@
+1	0	0	0	0
```

Comment:
I think the last column of zeros is because the `num_classes` parameter is set to `5`?

Reply:
Yes, this is a test where we have 4 classes in the input file, but num_classes is intentionally set to 5.